ui/src/app/pages/data/concept/concept-set-details.tsx

```diff
@@ -79,29 +94,47 @@ const ConceptSetMenu: React.FunctionComponent<{
   </PopupTrigger>;
 };
 
-export const ConceptSetDetails =
-  fp.flow(withUrlParams(), withCurrentWorkspace())(class extends React.Component<{
-    urlParams: any, workspace: WorkspaceData}, {
-      conceptSet: ConceptSet, deleting: boolean, editName: string, editDescription: string,
-      editSaving: boolean, error: boolean, errorMessage: string, editing: boolean,
-      loading: boolean, removingConcepts: boolean, removeSubmitting: boolean,
-      selectedConcepts: Concept[], workspacePermissions: WorkspacePermissions}> {
+export interface ConceptSetProps {
```

Comment:
Nit: I'd name these `ConceptSetDetailsProps` and state to match with component name.

Reply:
👍